src/runtime/composables/useSession.ts

```diff
@@ -189,20 +187,13 @@ const getSession = async (getSessionOptions?: GetSessionOptions) => {
       options.params = {
         ...(options.params || {}),
         // The request is executed with `server: false`, so window will always be defined at this point
-        callbackUrl: callbackUrl || getRequestURL()
+        callbackUrl: callbackUrl || callbackUrlFallback
```

Comment:
required as `onRequest` will also run asyncly later, with a non-existent nuxt instance in the context